common-moonmage.lic

```diff
@@ -320,4 +320,35 @@ module DRCMM
     DRC.message("Could not set planet data. Cannot cast #{data['abbrev']}")
   end
 
+  def set_moon_data(data, settings = nil)
```

Comment:
No sooner did I click merge than I realized something. Why does this function take `settings` as an arg? It doesn't seem to use it.